string name
string description
string cad
---
bool success

